src/components/theme.tsx

```diff
@@ -0,0 +1,179 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
```

Comment:
Is this file necessary to be created as `.tsx` ? i feel this can be a `.ts` file which can live in a different folder than components. For example: `./src/styles` ? In case you want to leave it as a component, please rename the file as `Theme.tsx`

Reply:
Not really agree with that. All files that contain JSX elements should be TSX while pure Typescript files should be TS.

Reply:
right, this is why i asked first to rename it to ".ts" . Anyway i left the chance to rename it as component because i was not sure if sveta was planning to introduce JSX code in it. In case she does not, then rename it to ".ts" is the right thing to do.

Reply:
yeap - my mistake

Reply:
done, it's .ts now